develop/devguide/GQI_Extensions/API_Reference/IGQIUpdater.md

```diff
@@ -19,7 +19,10 @@ It allows adding, removing, and updating rows, or updating individual cells.
 
 ### void AddRow(GQIRow)
 
-Adds a new row to the query result. If a row with the same row key already exists, that row will be updated instead. This method behaves as an *add-or-update* operation.
+Adds a new row to the query result. 
+
+> [!NOTE]
+> As of DataMiner main release 10.5.0-CU4 and feature release 10.5.7, if a row with the same row key already exists, that row will be updated. In other words, this method now behaves as an *add-or-update* operation. Prior to that, this would result in a duplicate row with the same row key to be added.
```

Comment:
I wonder if the note should explain how it worked prior to those versions, so that it correctly describes how it is on the latest version.

Reply:
Good point, applied your suggestion